cli/common/tarantool.go

```diff
@@ -327,13 +327,13 @@ func GetMajorCartridgeVersion(conn *connector.Conn) (int, error) {
 		return 0, err
 	}
 
-	if cartridgeVersionStr == "scm-1" {
+	if cartridgeVersionStr == "scm-1" || cartridgeVersionStr == "unknown" {
```

Comment:
By using git blame, you can trace that `unknown` version could be retrieved from the latest `cluster` rock and from any version of `cartridge`, including 1.x ones:

https://github.com/tarantool/cartridge/commit/038a163f14a76a5fdc962d28b1e9d4460ce72245

So cartridgeVersionStr == "unknown" may mean `1, nil` and `2, nil`.

Reply:
Ok, then let's do the error message more clear